keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -33,10 +35,39 @@ import org.springframework.context.annotation.Primary
 import org.springframework.core.Ordered.HIGHEST_PRECEDENCE
 import org.springframework.scheduling.TaskScheduler
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+private const val IPC_SERVER_METRIC = "controller.invocations"
 
 @EnableFiatAutoConfig
 @Configuration
-class DefaultConfiguration {
+class DefaultConfiguration(
+  val spectatorRegistry: Registry
+) : WebMvcConfigurer {
+
+  /**
+   * Enable controller metrics
+   */
+  override fun addInterceptors(interceptorRegistry: InterceptorRegistry) {
```

Comment:
Does the parent's implementation add any interceptors (logging, auth, perhaps)? Wondering if we should call `super.addInterceptors(interceptorRegistry)` at the end of this method.

Reply:
the parent is an interface with all the hook points having default noop implementations